packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/exampleSearch.e2e.ts

```diff
@@ -0,0 +1,151 @@
+import {testSearch, expect} from './fixture';
+
+let test = testSearch;
+
+test.describe('Example Search Page E2E Tests', () => {
+  test.describe('when loading the search page', () => {
+    test('should render correctly and load results automatically', async ({
+      searchPage,
+    }) => {
+      expect(searchPage.searchbox).toBeVisible();
+      expect(searchPage.facetsManager).toBeVisible();
+      expect(searchPage.facets).toBeVisible();
+      expect(searchPage.summary).toBeVisible();
+      expect(searchPage.sort).toBeVisible();
+      expect(searchPage.tabBar).toBeVisible();
+      expect(searchPage.pager).toBeVisible();
+      expect(searchPage.foldedResultList).toBeVisible();
+      expect(searchPage.recentQueriesList).toBeVisible();
+      expect(searchPage.recentResultsList).toBeVisible();
+
+      const firstResult = searchPage.results.first();
+      expect(firstResult).toBeVisible();
+      expect(await firstResult.textContent()).not.toBeNull();
+
+      const resultsCount = searchPage.results.count();
+      expect(await resultsCount).toBeGreaterThan(0);
+    });
+  });
+
+  test.describe('the happy path', () => {
+    test('should allow the user to search, sort, change the result page, change the tab, select a facet and clear filters', async ({
+      searchPage,
+      search,
+    }) => {
+      // Trigger a search query
+      const exampleQuery = 'test';
+      const searchRequestPromise = search.waitForSearchRequest();
+
+      await searchPage.searchboxInput.fill(exampleQuery);
+      await searchPage.searchboxInput.press('Enter');
+
+      await searchRequestPromise;
+      const searchRequestBody = (await searchRequestPromise).postDataJSON();
+      expect(searchRequestBody?.q).toBe(exampleQuery);
+
+      await expect(searchPage.summary).toContainText(exampleQuery);
+
+      const recentQueriesListItems =
+        await searchPage.recentQueriesListItems.all();
+      expect(recentQueriesListItems.length).toEqual(1);
+      const expectedQueries = [exampleQuery];
+      expectedQueries.forEach(async (expectedQuery: string, index: number) => {
+        // eslint-disable-next-line no-await-in-loop
+        await expect(recentQueriesListItems[index]).toHaveText(expectedQuery);
+      });
```

Comment:
There's only one expectedQuery in your array.
And you already test that the recentQueriesListItems is of length 1 only.

```suggestion
      const expectedQuery = exampleQuery;
      await expect(recentQueriesListItems[0]).toHaveText(expectedQuery);
```